docs/conf.py

```diff
@@ -58,6 +58,32 @@
 # The master toctree document.
 master_doc = 'index'
 
+# General information about the project.
```

Comment:
It looks like this is the info for OpenFAST, but within the WEIS conf.py. Is this on purpose?